server/jobs/offboarding/email.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-param-reassign */
+import { sendITOffboardingMail, sendSOPOffboardingMail } from '../../modules/email/emailModule';
+
+/**
+ * @desc Automates developer offboarding on slack
```

Comment:
This is `email` offboarding and not `slack` offboarding.

Reply:
Thanks for the point out